libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -78,6 +78,8 @@ export interface IAppHeaderProps {
 
     disableHelpDropdown?: boolean;
     onHelpClick?: (isOpen: boolean) => void;
+
+    redirectUrl?: string;
```

Comment:
The name should mention this is used in help only. How about st like `helpRedirectUrl`?